### PR TITLE
Updated Ubuntu 18.04 to latest

### PR DESCRIPTION
**Pull request by wills-int**
### Description 
_Add a comprehensive description of proposed changes_

**I UPDATED THE CI FROM UBUNTU-18.04 TO UBUNTU-LATEST AND RAN A TEST AND IT PAST SUCCESSFULLY**

Fixes # - _issue number(s) if exists_

- [ ] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [x] infrastructure - _change in infrastructure and CI_
- [ ] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

**Comment by isaevil** [comment]
@dessie-williams was there an issue with Ubuntu 18.04?

**Comment by wills-int** [comment]
> @dessie-williams was there an issue with Ubuntu 18.04?

@isaevil This was a request, put in by Timmie, to make the changes to the script and test the changes. The 18.04 will not be supported, after March.
[PREINFRA-772](https://jira.devtools.intel.com/browse/PREINFRA-772)

File: .github/workflows/ci.yml
Comment (isaevil) [inline .github/workflows/ci.yml:1]: Please, update copyright year 2022->2023

Comment (wills-int) [reply .github/workflows/ci.yml:1]: @isaevil Changes have been done.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2022 Intel Corporation
+# Copyright (c) 2022-2023 Intel Corporation
```
Comment (isaevil) [inline .github/workflows/ci.yml:1]: It should be 2021-2023

**Review by isaevil — CHANGES_REQUESTED** [review]

**Review by isaevil — APPROVED** [review]

**Pull request merged**